src/WebJobs.Script/Workers/WorkerHelpers.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net;
+using System.Net.Sockets;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers
+{
+    public static class WorkerHelpers
```

Comment:
Nit: rename WorkerHelpers to WorkerUtilities to follow similar naming convention as existing helper classes